package.json

```diff
@@ -4,10 +4,10 @@
   "version": "2.0.1",
   "main": "index.js",
   "dependencies": {
-    "eslint-plugin-ember": "7.x.x",
+    "eslint-plugin-ember": "8.x.x",
```

Comment:
The last version is now 9. Should it be updated ?

Reply:
Ping @GreatWizard .

Reply:
Indded thanks you for checking :)